Web/Startup.cs

```diff
@@ -91,6 +91,11 @@ private void ConfigureStandardServices(IServiceCollection services)
             options.MinimumSameSitePolicy = SameSiteMode.None;
         });
 
+        services.AddAntiforgery(options =>
+        {
+            options.Cookie.SecurePolicy = CookieSecurePolicy.SameAsRequest;
```

Comment:
Any reason we don't want to do CookieSecurePolicy.Always instead?

Reply:
Yes, it can make it a bit cumbersome to run the project locally. 
CookieSecurePolicy.Always essentially makes https required to run the application - also locally. SameAsRequest allows running both behind http and https, but since prod is always behind https it's not really a problem.

Reply:
Yes, it can make it a bit cumbersome to run the project locally.
CookieSecurePolicy.Always essentially makes https required to run the application - also locally. SameAsRequest allows running both behind http and https, but since prod is always behind https it's not really a problem.